pkg/sql/logictest/tests/local/local_test: TestLogic_lookup_join_local failed

pkg/sql/logictest/tests/local/local_test.TestLogic_lookup_join_local [failed](https://tanzanite.cluster.engflow.com/invocations/default/389e6840-3599-499a-b0e2-9cd00230a383?testReportRun=1&testReportShard=29&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [e2e88aa288c81282b20ca0640da983f97b234324](https://github.com/cockroachdb/cockroach/commits/e2e88aa288c81282b20ca0640da983f97b234324):

Fatal error:

```
panic: test timed out after 1h29m52s
running tests:
	TestLogic_lookup_join_local (1h28m43s)

```
Stack: 

```
goroutine 811864 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2259 +0x259
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestLogic_lookup_join_local
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestLogic_lookup_join_local1232692394
    test_log_scope.go:81: use -show-logs to present logs inline
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=29</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_lookup_join_local.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34822

pkg/sql/logictest/tests/local/local_test.TestLogic_lookup_join_local [failed](https://tanzanite.cluster.engflow.com/invocations/default/8d5b8c67-35b6-4168-965f-87485669da8d?testReportRun=2&testReportShard=29&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [ff291e3ee70eedac50b985860502d2f0b0867003](https://github.com/cockroachdb/cockroach/commits/ff291e3ee70eedac50b985860502d2f0b0867003):

Fatal error:

```
panic: test timed out after 1h29m52s
running tests:
	TestLogic_lookup_join_local (1h27m59s)

```
Stack: 

```
goroutine 878774 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2259 +0x259
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestLogic_lookup_join_local
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestLogic_lookup_join_local3407854369
    test_log_scope.go:81: use -show-logs to present logs inline
[10:20:39] setting distsql_workmem='90657B';
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=29</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_lookup_join_local.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/generate-logictest/templates.go
+++ b/pkg/cmd/generate-logictest/templates.go
@@ -52,6 +52,9 @@ const templateText = `
 {{- if .LogicTest -}}
 func runLogicTest(t *testing.T, file string) {
 	{{ if .Is3NodeTenant }}skip.UnderRace(t, "large engflow executor is overloaded by 3node-tenant config")
+	{{ else if eq .TestRuleName "local"}}if file == "lookup_join_local" {
+		skip.UnderRace(t, "this file is too slow under race")
+	}
 	{{ end }}skip.UnderDeadlock(t, "times out and/or hangs")
 	logictest.RunLogicTest(t, logictest.TestServerArgs{}, configIdx, filepath.Join(logicTestDir, file))
 }

--- a/pkg/sql/logictest/tests/local/generated_test.go
+++ b/pkg/sql/logictest/tests/local/generated_test.go
@@ -60,6 +60,9 @@ func TestMain(m *testing.M) {
 }
 
 func runLogicTest(t *testing.T, file string) {
+	if file == "lookup_join_local" {
+		skip.UnderRace(t, "this file is too slow under race")
+	}
 	skip.UnderDeadlock(t, "times out and/or hangs")
 	logictest.RunLogicTest(t, logictest.TestServerArgs{}, configIdx, filepath.Join(logicTestDir, file))
 }

